examples/README.md

```diff
@@ -18,6 +18,21 @@ some of the most popular database libraries and frameworks.
 - [SQL Server (ESM)](./knex/tedious/connect.mjs)
 - [SQL Server (TypeScript)](./knex/tedious/connect.ts)
 
+### Sequelize
+
+> [!WARNING]
+> Sequelize examples require Sequelize v7.0.0-alpha.27 (`npm install @sequelize/core`)
```

Comment:
@ruyadorno do they require the exact version or just greater than?

Reply:
greater than! I'll add it here 👍